tcs-client/pom.xml

```diff
@@ -37,7 +37,7 @@
     <dependency>
       <groupId>com.transformuk.hee.tis</groupId>
       <artifactId>tcs-api</artifactId>
-      <version>6.14.0</version>
+      <version>6.14.1</version>
```

Comment:
The `PlacementValidator` is in tcs-service.

Reply:
tcs-api 6.14.1 has PlacementDetailsDTO with the new `esrEvents` field.
tcs-api 6.14.0 has PlacementDetailsDTO without the new field.

Generic Uploads has a dependency on tcs-client, which has a dependency on tcs-api. If tcs-client isn't updated with the latest dependency on tcs-api, the PlacementDetailsDTOs used in Generic Upload won't have the new `esrEvents` field.

It took me a while to realize yesterday cos I couldn't figure out where in Generic Upload tcs-api was being pulled in. Without this dependency upgrade the PlacementDetailsDTOs sent to TCS by Generic Uplaod won't have the `esrEvents` field filled.

Reply:
Yeah, you're right. This one is for using new PlacementDetailsDTO in Bulk upload. 
Apart from this, the version of tcs-service also needs a bump up.